substrate/frame/bounties/src/lib.rs

```diff
@@ -460,8 +460,7 @@ pub mod pallet {
 
 				let slash_curator = |curator: &T::AccountId,
 				                     curator_deposit: &mut BalanceOf<T, I>| {
-					let imbalance = T::Currency::slash_reserved(curator, *curator_deposit).0;
-					T::OnSlash::on_unbalanced(imbalance);
```

Comment:
`pallet_bounty` still rely on the old low-level implementation `Propose > Approve` so I wonder if removing this `on_unbalanced` when `slash_reserved` affects the existing use of the pallet.